core/src/main/resources/error/error-classes.json

```diff
@@ -1046,6 +1046,63 @@
       "Protobuf type not yet supported: <protobufType>."
     ]
   },
+  "PYSPARK" : {
```

Comment:
Later, when the amount of error classes becomes large enough to be categorized, it can be subdivided into several error classes starting with PYSPARK.

e.g. "PYSPARK_INVALID_TYPE", "PYSPARK_WRONG_NUM_ARGS" or something.